Libs/MRML/Core/vtkMRMLScalarVolumeDisplayNode.cxx

```diff
@@ -796,7 +796,7 @@ void vtkMRMLScalarVolumeDisplayNode::CalculateAutoLevels()
 
     window = max-min;
     level = 0.5*(max+min);
-    lower = this->GetLevel();
+    lower = range[0];
```

Comment:
@lassoan I think the level was chosen as the default lower threshold because it is used when making a slice model visible in the threeDView and in the paradigm use case you'd want to see bright tissue while making the air transparent.  Since thresholding is off at initialization you are right the default value has little impact.